packages/vkui/src/components/FormItem/FormItem.e2e-playground.tsx

```diff
@@ -47,12 +51,31 @@ export const FormItemPlayground = (props: ComponentPlaygroundProps) => {
         {
           top: ['Сверху'],
           bottom: ['Снизу'],
+          bottomId: ['bottom-id'],
           removable: ['indent'],
-          children: [<Input key={0} placeholder="Введите ваше значение" />],
+          children: [
+            <Input key={0} aria-labelledby="bottom-id" placeholder="Введите ваше значение" />,
+          ],
         },
       ]}
     >
-      {(props: FormItemProps) => <FormItem {...props} />}
+      {({
+        bottom,
+        bottomId: bottomIdProp,
+        children: childrenProp,
+        ...restProps
+      }: FormItemProps) => {
+        const bottomId = bottom && !bottomIdProp ? 'bottom-id' : bottomIdProp;
+        const children =
+          bottomId && typeof childrenProp === 'function'
```

Comment:
```suggestion
          typeof childrenProp === 'function'
```

скриншоты поломались, давай всегда вызывать функцию